src/lib/components/CopyCode/index.js

```diff
@@ -22,17 +22,22 @@ class CopyCode extends BaseElement {
     if (!this.copyButton) {
       this.copyButton = document.createElement('button');
       this.copyButton.className =
-        'w-button--icon w-button--round web-copy-code__button';
+        'icon-button icon-button__round web-copy-code__button';
       // Set aria-label because title isn't accessible to sighted keyboard users
       // and the tooltip is only visible on focus,
       // which means it isn't read reliably by screen readers.
       this.copyButton.setAttribute('aria-label', 'Copy code');
       this.copyButton.addEventListener('click', this.onCopy);
 
       this.tooltip = document.createElement('span');
-      this.tooltip.className = 'w-tooltip w-tooltip--right';
-      this.tooltip.setAttribute('role', 'tooltip');
-      this.tooltip.textContent = 'Copy code';
+      this.tooltip.className = 'tooltip';
+      this.tooltip.setAttribute('data-alignment', 'right');
```

Comment:
I think we should keep `this.tooltip.setAttribute('role', 'tooltip');`. This is needed for proper a11y.